.github/workflows/checks.yml

```diff
@@ -5,6 +5,7 @@ on:
       - .github/**
   push:
     branches:
+      dev
       main
```

Comment:
I've initially considered using `prod` branch for prod and `main` for development, but this meant that the live site URL would change from `main.___.amplifyapp.com` to `prod.___.amplifyapp.com` so opted to use `dev` for dev and `main` for prod.

Reply:
Thanks for clarifying that, because I don't know how the AWS Amplify stuff was setup.